purchase_request_to_procurement/models/purchase_request.py

```diff
@@ -73,6 +73,7 @@ def _prepare_procurement_order_values(
             'product_qty': self.product_qty,
             'product_uom': self.product_uom_id.id,
             'date_planned': self.date_required,
+            'request_id': self.request_id.id,
```

Comment:
The purchase request lines have a field `procurement_id` that unequivocally gives you the origin. While in the procurement order the link is with the last purchase request created/modified where the line was added, this is like this just to follow the same behavior as with purchase orders (the link in the procurement is to a PO not a PO line).

For your second point, I had already realized of that, but If you use this module alone without `purchase_request_procurement` the only issue is that in the logs a warning would appear saying that the value 'request_id' is useless (kind of, don't know now the exact msg :wink:). But when you use this module in conjunction with `purchase_request_procurement` it makes sense. I rather prefer this than to add a new dependency just for this.

What do you think?